app/controllers/application_controller.rb

```diff
@@ -5,7 +5,11 @@ def current_user
     @current_user ||= User.find(session[:user_id]) if session[:user_id]
   end
 
+  def current_admin
+    true if current_user && current_user.admin?
+  end
+
```

Comment:
This can be shortened to just `current_user && current_user.admin?` (that evaluates to true if both are true and false otherwise)